tutorials/First_Data_Fetching.ipynb

```diff
@@ -7,17 +7,22 @@
    "source": [
     "# Tutorial One - Data Preparation\n",
     "\n",
-    "This tutorial will give an overview of two methods of preparing sample data for use in the tutorial notebooks which demonstrate the use of each scoring function. The result of this tutorial will be two files - \"forecast_grid.nc\" and \"analysis_grid\".nc. The two methods covered will be (a) downloaded data from the Austration National Computational Infrastructure, and (b) synthetic data. \n",
+    "This tutorial gives an overview of two methods of preparing sample data for use in the tutorial notebooks demonstrating the use of each scoring function. The result of this tutorial will be two files - \"forecast_grid.nc\" and \"analysis_grid.nc\". The two methods covered will be \n",
     "\n",
-    "Numerical weather prediction data is large in size, and comes in many formats and structures. Common structures will include:\n",
+    "1. data download from the Austration National Computational Infrastructure, \n",
+    "2. generation of synthetic data. \n",
+    "\n",
+    "Weather prediction data is typically large in size, and comes in many formats and structures. Common structures will include:\n",
     "\n",
     " - a single value for a prediction such as predicted maximum temperature (e.g. \"The forecast for your town tomorrow is a maximum of 25 degrees\"); \n",
     " - a data array representing that value across a geographical area (such as a region, a country or the whole planet); \n",
     " - a data array representing multiple possible predicted values (arising from ensemble predictions), or an expected value and confidence intervals. \n",
-    "  \n",
-    "The data in these tutorials will start with typical model data outputs and ensembles. This will be made clearer with an example.\n",
     "\n",
-    "Downloaded data is usually covered by a license agreement. You should check the license agreements for the downloaded data in these examples."
+    "There is often also a time dimension to the data.\n",
+    "\n",
+    "The data in the `scores` tutorials starts with typical numerical model data outputs and ensembles. This will be made clearer with an example.\n",
+    "\n",
+    "Downloaded data is usually covered by a license agreement. "
```

Comment:
I think it's important we retain the comment that users should check the license themselves. I don't want to make assumptions and it's a good reminder.

Reply:
I'm fine with that. I'll remove that comment.